ee/tabby-webserver/src/service/answer.rs

```diff
@@ -492,6 +490,17 @@ Remember, don't blindly repeat the contexts verbatim. When possible, give code s
     )
 }
 
+// https://platform.openai.com/docs/api-reference/chat/create#chat-create-presence_penalty
```

Comment:
this won't be very useful - since the backend implementation might be another api (e.g codestral, deepseek, ...).

I recommend just pass this to backend model as is.